aiida/scheduler/plugins/test_pbspro.py

```diff
@@ -908,6 +908,7 @@ def test_submit_script(self):
         s = PbsproScheduler()
 
         job_tmpl = JobTemplate()
+        job_tmpl.shebang = '#!/bin/bash -l'
```

Comment:
You put a `-l` as a way to verify the current implementation?
Maybe if you implement also the case with `shebang` not specified, you can have a copy of one of these tests to verify that even if you don't set it, you get the default value?